freeipa/src/main/java/com/sequenceiq/freeipa/sync/StackStatusCheckerJob.java

```diff
@@ -100,11 +100,16 @@ public void syncAStack(Stack stack) {
                     }
                     if (!checkableInstances.isEmpty()) {
                         SyncResult syncResult = freeipaChecker.getStatus(stack, checkableInstances);
-                        List<ProviderSyncResult> results = providerChecker.updateAndGetStatuses(stack, checkableInstances, syncResult.getInstanceStatusMap());
-                        if (!results.isEmpty()) {
-                            updateStackStatus(stack, syncResult, results, alreadyDeletedCount);
+                        if (DetailedStackStatus.AVAILABLE == syncResult.getStatus()) {
```

Comment:
I think I know the answer to this but I wanted to check. Is the reason that `DetailedStackStatus.UNHEALTHTY` can't bypass the cloud provider check because we need to catch the case where 1 VM is running and 1 VM is either stopped, deleted, etc?

Reply:
DetailedStackStatus.AVAILABLE happens only when freeipa in all instances are available. That is skipped by this change.